src/App.tsx

```diff
@@ -16,35 +16,92 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [isSortedAlphabetically, setIsSortedAlphabetically] = useState<boolean>(false);
+  const [isSortedByLength, setIsSortedByLength] = useState<boolean>(false);
```

Comment:
Create 1 state for `sortType` that is applied 
Create `enum` for this `sortType` and use it as a value of your state